system/lib/libc/emscripten_scan_stack.c

```diff
@@ -0,0 +1,8 @@
+#include <emscripten.h>
```

Comment:
do we need both of these includes?

Reply:
Yeah .. this function should really be declared in "stack.h" but its in "emscripten.h" proper.. unlike the two function I'm calling which are from "stack.h".